FixDevV3Blobs/FixDevV3Blobs/Program.cs

```diff
@@ -0,0 +1,279 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Storage;
+using Microsoft.Azure.Storage.Blob;
+
+namespace FixDevV3Blobs
+{
+    class Program
+    {
+        const int MaxTasks = 32;
```

Comment:
It would be great to add a ReadMe file with the tool description and command line args.